routes/index.js

```diff
@@ -24,11 +25,16 @@ const constructorMethod = (app) => {
       universityName = university.name;
       itemsList = await items.getAllByUniversityId(user.universityId);
     }
+    else{
+      let universityCollection = await universities.getAll();
+      universityCollection.forEach(university => universityList.push(university.name));
```

Comment:
I think you can do `universityList = await universities.getAll();` and you won't need line 30.